### ISSUE TITLE
Report failed

### ISSUE THREAD

**Issue by fetyj** [fmalmeida/bacannot#66]
Hi Felipe,
Could you help with this issue I've got when running bacannot o my genomes, everything went just fine except the report step. I tried with my old genomes (which were successful at the time) and I get the same error with the last step (report).
```
Error executing process > 'BACANNOT:REPORT (S6272)'

Caused by:
  Process `BACANNOT:REPORT (S6272)` terminated with an error exit status (1)

Command executed:

  #!/usr/bin/env Rscript
  
  ## Copy reports
  system("cp /work/reports/* .") ;
  
  ## Remove empty files
  system("rm -f input.??") ;
  system("rm -f input.?") ;
  
  ## Generate generic Report
  rmarkdown::render("report_general.Rmd" ,   params = list( prokka  = "S6272.txt",                  kegg    = "S6272_kegg-decoder_heatmap_static.svg",                  barrnap = "S6272_rRNA.gff",                  mlst    = "S6272_mlst_analysis.txt",                  refseq_masher = "refseq_masher_results.txt",                  query = "S6272")) ;
  
  ## Generate Resistance Report
  rmarkdown::render("report_resistance.Rmd", params = list(    blast_id = 90 ,     blast_cov = 90,     amrfinder = "AMRFinder_resistance-only.tsv",     query = "S6272",     rgitool = "RGI_S6272.txt",     rgiparsed = "Parsed_RGI_S6272_hits.txt",     rgi_heatmap = "input.10",     argminer_blastp = "S6272_argminer_blastp_onGenes.summary.txt",     resfinder_tab = "input.12",     resfinder_pointfinder = "input.13",     resfinder_phenotype = "input.14",     gff = "S6272.gff")) ;
  
  ## Generate Virulence Report
  rmarkdown::render("report_virulence.Rmd" ,   params = list( blast_id = 90 ,                  blast_cov = 90,                  vfdb_blast = "S6272_vfdb_blastn_onGenes.txt",                  gff = "S6272.gff",                  victors_blast = "S6272_victors_blastp_onGenes.txt",                  query = "S6272")) ;
  
  ## Generate MGEs report
  rmarkdown::render("report_MGEs.Rmd",   params = list( blast_id = 85,                  blast_cov = 85,                  phigaro_dir = "/choppy/Bacteria_assembly/Lb_crispatus_S6272_bacannot/prophages/phigaro",                  phigaro_txt = "S6272_phigaro.tsv",                  phispy_tsv = "prophage.tsv",                  ice_prot_blast = "S6272_iceberg_blastp_onGenes.txt",                  ice_genome_blast = "S6272_iceberg_blastn_onGenome.summary.txt",                  plasmid_finder_tab = "results_tab.tsv",                  platon_tsv = "S6272.tsv",                  query = "S6272",                  gi_image = "teste.png",                  digis = "transposable_elements_digis.gff",                  gff = "S6272.gff",                  phast_prot_blast = "S6272_phast_blastp_onGenes.txt" )) ;

Command exit status:
  1

Command output:
  (empty)

Command error:
  Error in knit_params_get(input_lines, params) : 
    render params not declared in YAML: prokka
  Calls: <Anonymous> -> knit_params_get
  Execution halted

Work dir:
  /choppy/work/b4/6bf748098f0a65f87fc585bd51071a

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`
```

Thanks,

Fety

**Comment by fmalmeida** [comment]
Hi @fetyj ,

thanks for using the pipeline.
I think this is a simple error. In latest versions, I had to update the reports, so they can be compliant with bakta as well.

Unfortunately, when doing that, I just realised later that it broke previous v3.1 versions as they expect a different manner of rendering.

As I saw that after publishing, I still don’t know how to properly fix it.

But, if you use the latest versions of docker images and the pipeline itself, this error should not happen.
